libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -888,11 +888,8 @@ bool GCS_MAVLINK::send_gps_raw(AP_GPS &gps)
         return false;
     }
 
-    if (gps.highest_supported_status(0) > AP_GPS::GPS_OK_FIX_3D) {
-        if (HAVE_PAYLOAD_SPACE(chan, GPS_RTK)) {
-            gps.send_mavlink_gps_rtk(chan);
-        }
-
+    if (HAVE_PAYLOAD_SPACE(chan, GPS_RTK)) {
```

Comment:
Plesa take a look at pymavlink#88 . Does it really hurt you to have GPS2_RTK active when you have a RTK GPS ?
I do not think that a enable parameter is needed here.